.github/workflows/release.yaml

```diff
@@ -219,7 +223,7 @@ jobs:
 
       - name: Run Docker and Extract Artifacts
         run: |
-          id=$(docker create build-deb:${{ github.run_id }})
+          id=$(docker create ${{ env.TAG }})
           docker cp $id:/build/${{ github.job }}_${{ needs.validate-preconditions.outputs.version }}.deb ./${{ env.ARTIFACT_NAME }}
 
       - name: Upload artifact
```

Comment:
Also remember to upload docker image here with the id of docker-p2p-bootstrapper